protobuf/coreapi/api.proto

```diff
@@ -402,6 +405,21 @@ message ServiceLogsRequest {
   repeated string dependencies = 2; // __Optional.__ List of dependencies to filter service logs. All by default.
 }
 
+// The request to fetch all informations of the Core
+message InfoRequest {}
+
+// Information concerning the Core from the `info` API.
+message InfoReply {
+  message CoreService {
+    string sid = 1;
+    string hash = 2;
+    string url = 3;
+  }
+  repeated CoreService services = 1;
+  string Address = 2;
+  string Image = 3;
```

Comment:
Replace image by version

Reply:
why ? ok to add another field version but this is the docker image, it's not the version. Even if there is a link between the version and the docker image let's not mix them.